src/Sql/dbo/Stored Procedures/Provider_ReadAbilities.sql

```diff
@@ -0,0 +1,12 @@
+﻿CREATE PROCEDURE [dbo].[Provider_ReadAbilities]
+AS
+BEGIN
+    SET NOCOUNT ON
+
+    SELECT
+        [Id],
+        [UseEvents],
+        [Enabled]
+    FROM
+        [dbo].[Provider]
```

Comment:
Would we want `SELECT` action to go against the `[dbo].[ProviderView]` instead of the underlying table?

Reply:
I based it on the `Organization_ReadAbilities` which uses `[dbo].[Organization]`.